nav-app/src/app/layer-upgrade/layer-upgrade.component.html

```diff
@@ -0,0 +1,191 @@
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+<div class="layer-upgrade">
+    <mat-card>
+        <mat-card-content>
+            <!-- title -->
+            <span class="title">
+                <h2>Layer Upgrade</h2>
+                ({{compareTo.version}} <mat-icon inline="true">east</mat-icon> {{viewModel.version}})
+            </span>
+            <!-- filter -->
+            <div>
+                <input id="filter" class="checkbox-custom" type="checkbox" [(ngModel)]="showAnnotatedOnly">
+                <label for="filter" class="checkbox-custom-label noselect">show annotated techniques only</label>
+            </div>
+        </mat-card-content>
+    </mat-card>
+
+    <mat-divider></mat-divider>
+
+    <!-- changelog -->
+    <div class="changelog">
+        <mat-accordion class="headers-align">
+        <mat-expansion-panel *ngFor="let section of sections" [disabled]="!sectionLength(section)">
+            <mat-expansion-panel-header>
+                <mat-panel-title>
+                    <h4>{{getHeader(section)}}</h4>
+                </mat-panel-title>
+                <mat-panel-description>
+                    ({{sectionLength(section)}}/{{changelog[section].length}} techniques)
+                </mat-panel-description>
+            </mat-expansion-panel-header>
+
+            <ng-template matExpansionPanelContent> <!-- changelog lazy render -->
+        
+            <!-- select all objects in section -->
+            <div class="checkbox">
+                <input id="select_{{section}}" class="checkbox-custom" type="checkbox"
+                    [checked]="allReviewed(section)" (click)="reviewAllChanged(section)">
+                <label for="select_{{section}}" class="checkbox-custom-label noselect">select all</label>
+            </div>
+
+            <!-- techniques -->
+            <ng-container *ngFor="let attackID of changelog[section]">
+                <mat-expansion-panel *ngIf="!showAnnotatedOnly || anyAnnotated(attackID)">
+                    <mat-expansion-panel-header>
+                        <mat-panel-title>
```

Comment:
The content of the header doesn't seem to be center aligned. Some elements are above or below the midline and it's a bit inconsistent between elements as well:
<img width="399" alt="Screen Shot 2021-07-12 at 1 04 20 PM" src="https://user-images.githubusercontent.com/28868888/125328073-e1f7e180-e311-11eb-8684-c3f06a4f04e2.png">